src/main/java/org/folio/clients/InventoryClient.java

```diff
@@ -40,6 +43,8 @@
 import static org.folio.util.ExternalPathResolver.MATERIAL_TYPES;
 import static org.folio.util.ExternalPathResolver.resourcesPathWithPrefix;
 
+import javax.swing.text.html.Option;
```

Comment:
Why do we need this class from 'javax.swing' package? Can we use something else instead?

Reply:
I think it's not used import, I will remove it, thanks